niceml/utilities/ioutils.py

```diff
@@ -192,6 +193,28 @@ def write_json(
         json.dump(data, file, **kwargs)
 
 
+def write_chart(
+    chart: Chart,
+    filepath: str,
+    file_system: Optional[AbstractFileSystem] = None,
+    file_format: Optional[Literal["json", "html", "png", "svg", "pdf"]] = "html",
+    **kwargs,
+):
+    """
+    Writes altair chart to filepath with optional AbstractFileSystem
```

Comment:
Suggestion: "Writes a altair chart to a file path. An AbstractFileSystem can be specified optionally. Otherwise LocalFileSystem is used by default."